src/fights.ts

```diff
@@ -2096,6 +2097,7 @@ export function deliverThesisIfAble(): void {
   safeRestore();
 
   if (
+    myBuffedstat($stat`Muscle`) < 1800 &&
```

Comment:
Matches the muscle stat lower down. Should I extract to a local `const`?
https://github.com/Loathing-Associates-Scripting-Society/garbage-collector/blob/acf0323471e4fd54c6e028db77faa0718c839426/src/fights.ts#L2110

Reply:
yeah, good shout

Reply:
Cool, going to refactor a lot more. Based on the comment about maximising muscle before triple-sized.